classes/workflow/EditorDecisionActionsManager.inc.php

```diff
@@ -19,6 +19,7 @@
 // Submission and review stages decision actions.
 define('SUBMISSION_EDITOR_DECISION_ACCEPT', 1);
 define('SUBMISSION_EDITOR_DECISION_DECLINE', 4);
+define('SUBMISSION_EDITOR_DECISION_INITIAL_DECLINE', 9);
```

Comment:
Since this constant is referred to in `pkp-lib`, some of this will also need to be ported to OMP so that we don't get "undeclared constant" warnings there.